src/drivers/px4io/px4io.cpp

```diff
@@ -996,19 +999,14 @@ int PX4IO::io_handle_status(uint16_t status)
 	/**
 	 * Get and handle the safety status
 	 */
-	const bool safety_off = status & PX4IO_P_STATUS_FLAGS_SAFETY_OFF;
+	const bool trigger_safety_button = status & PX4IO_P_STATUS_FLAGS_SAFETY_OFF;
```

Comment:
IMO the previous name was more appropriate, describing a state, not a state change.